src/FlatbuffersTranslator.cpp

```diff
@@ -90,6 +91,7 @@ FlatbuffersTranslator::createParser(const std::string &FullName,
                                     const std::string &Message,
                                     const std::string &Schema) {
   flatbuffers::IDLOptions opts;
+  opts.strict_json = true;
```

Comment:
It would be nice to have an inline comment here to say what happens if you don't set this.